Makefile

```diff
@@ -32,9 +32,6 @@ test:
 clean:
 	rm -f $(MICROSERVICES)
 
-update:
-	$(GO) mod download
-
 docker: $(DOCKERS)
```

Comment:
Remove

Reply:
is there something that needs to be removed?

Reply:
Remove line 35 once target below has been renamed to `docker`